models/utils/marketo__calendar_spine.sql

```diff
@@ -1,18 +1,40 @@
+-- depends_on: {{ ref('stg_marketo__lead') }}
+-- above comment must be kept for dbt to run
+
 with spine as (
 
+    {% if execute %}
+    {% set first_date_query %}
+        select  min( created_timestamp ) as min_date from {{ ref('stg_marketo__lead') }}
```

Comment:
I think we should instead change the reference to point to the source `{{ source('marketo','lead') }}`. This way if the user runs `dbt compile` before `dbt run` they won't see an error.

See the updates we applied to the latest [Zendesk PR](https://github.com/fivetran/dbt_zendesk/pull/79) that addressed an issue like this.

Reply:
Done.